plutus-tx/test/Blueprint/Definition/Fixture.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeApplications      #-}
+{-# LANGUAGE UndecidableInstances  #-}
+{-# LANGUAGE ViewPatterns          #-}
+
+module Blueprint.Definition.Fixture where
```

Comment:
I'm not really sure what "Fixture" means here and elsewhere. Maybe it's some standard terminology that I'm unaware of?

Reply:
Yep, https://en.wikipedia.org/wiki/Test_fixture